bundles/ui/org.eclipse.smarthome.ui.basic/README.md

```diff
@@ -0,0 +1,13 @@
+## Eclipse Smarthome BasicUI
+
+BasicUI is a web interface for Eclipse Smarthome based on Material Design Lite from Google.
+
+## Contributing
```

Comment:
Please remove the contributing part from this README. You can create a CONTRIBUTING.md file for it, if you like.